samples/dnn/action_recognition.py

```diff
@@ -0,0 +1,81 @@
+import os
+import numpy as np
+import cv2 as cv
+import argparse
+from common import findFile
+
+parser = argparse.ArgumentParser(description='Use this script to run action recognition using 3D ResNet34')
+parser.add_argument('--input', '-i', help='Path to input video file. Skip this argument to capture frames from a camera.')
+parser.add_argument('--model', help='Path to model.')
+parser.add_argument('--classes', default=findFile('action_recongnition_kinetics.txt'), help='Path to classes list.')
```

Comment:
Please mark `model` as required (`required=True`) and add `formatter_class=argparse.ArgumentDefaultsHelpFormatter` to `ArgumentParser`:

```python
parser = argparse.ArgumentParser(description='Use this script to run action recognition using 3D ResNet34',
                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```
This way user could see default arguments (i.ei path to classes file).

Reply:
done